newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -440,6 +440,14 @@ export const HomePage = React.memo<Props>(
         [authenticated]
       );
 
+      const handleGameUpdated = React.useCallback(
+        (game: Game) => {
+          onGameUpdated(game);
+          fetchGames();
```

Comment:
why is this needed? shouldn't `onGameUpdate()` be in charge of fetching the games again if needed?

Reply:
I reused the logic that was there before:
One updates the game directly in the state.
The other one fetches the games.
We need to fetch games because there is a logic that sets the chosen game to null in order to show the loader and then the API call comes back and the chosen game is set back to the right one.
I'll look into it tomorrow with fresh eyes, maybe there's more straightforward